src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -217,8 +222,8 @@ using LayerTypes = boost::variant<
     BaseLayer<LogisticFunction, arma::mat, arma::mat>*,
     BaseLayer<IdentityFunction, arma::mat, arma::mat>*,
     BaseLayer<TanhFunction, arma::mat, arma::mat>*,
-    BaseLayer<RectifierFunction, arma::mat, arma::mat>*,
```

Comment:
Hey @saksham189 , @zoq is there any limitation in `boost::variant` we cannot give more than 50 parameters that's why my builds are failing but when I removed this I was successfully able to build.

Reply:
Hey @himanshupathak21061998 yes there is a limitation of 50. That is why we introduced `MoreTypes`. Can you try adding your layer under that?